envi/memory.py

```diff
@@ -462,7 +492,9 @@ def getMemoryMaps(self):
         return [mmap for mva, mmaxva, mmap, mbytes in self._map_defs]
 
     def readMemory(self, va, size):
-
+        '''
+        Read memory from maps stored in self._map_defs
```

Comment:
nit: `_map_defs` is a "private" field so probably shouldn't be referenced in public documentation.

Reply:
heh.  yep.  good call.  running 1000 miles a minute lately.